tests/lexer/lexer_test.cpp

```diff
@@ -1,13 +1,70 @@
 #include "catch2/catch.hpp"
 #include "lexer.hpp"
+#include <string>
+#include "fmt/format.h"
+#include "trie.hpp"
+#include "catch_helpers.hpp"
 
-SCENARIO("Summing two numbers") {
-    // By the way, don't mention constants in descriptions
-    GIVEN("Two positive numbers: 2 and 3") {
-        int a = 2, b = 3;
+using namespace std;
+namespace testing
+{
 
-        THEN("their sum is 5") {
-            REQUIRE(sum(a, b) == 5);
+    void checkInTrie(string key, common::Trie<int>
+                                     trie)
+    {
+        auto result = trie.Find(key);
+        CHECK_MESSAGE((bool)result,
+                      "expected to find key \"{}\", but didn't", key);
+    }
+} // namespace testing
+SCENARIO("Trie initialization")
+{
+
+    GIVEN("The file name")
+    {
+        THEN("it s readen")
+        {
+            REQUIRE(read() == 0);
+        }
+    }
+    GIVEN("A command to initialize the trie")
+    {
+        common::Trie<int> trie = initTrie();
+        THEN("All 24 keywords ar in the trie")
+        {
+            string keywords[]{"var", "type", "routine", "is", "integer",
+                              "real", "boolean", "record", "array", "true",
+                              "false", "while", "for", "loop", "end",
+                              "reverse", "in", "if", "else", "and",
+                              "or", "xor", "then", "return"};
+            for (string kw : keywords)
+            {
+                testing::checkInTrie(kw, trie);
+            }
+        }
+        THEN("Boolean ops are in the tire")
```

Comment:
```suggestion
        THEN("Boolean ops are in the trie")
```
Same typo in below tests